### DISCUSSION TITLE
Add django-debug-toolbar

### DISCUSSION THREAD

**Discussion by ShreehariVaasishta** [djangopackages/djangopackages#685] [General]
Using [django-debug-toolbar ](https://github.com/jazzband/django-debug-toolbar) in the development environment is very helpful and will surely lead to ways for improving the project.

**Comment by jefftriplett** [comment]
I noticed that we have it installed, but I didn't see it when I was playing around the other day. I wonder what is missing that's keeping it from showing up? 

https://github.com/djangopackages/djangopackages/search?q=toolbar

**Reply by ShreehariVaasishta** [reply]
I will look into it and make sure it is working again.

**Reply by smithdc1** [reply]
Looks like the middleware is commented out? https://github.com/djangopackages/djangopackages/blob/e39c6a65c1533c71c54c9486238067d70e4203fa/settings/base.py#L272

**Reply by jefftriplett** [reply]
I commented it out after seeing some testing errors. I need to try it again and write up a proper bug report.

**Reply by saadmk11** [reply]
This merged PR enabled debug-toolbar for local development
- https://github.com/djangopackages/djangopackages/pull/934

**Comment by ryancheley** [comment]
It looks as though `django-debug-toolbar` is enabled. I'm closing this discussion
